### ISSUE TITLE
Final report(s) and outputs delivered

### ISSUE THREAD

**Issue by nukeador** [mozilla/participation-metrics-org#106]

**Comment by jgbarah** [comment]
Not sure if this is the right place to discuss this. However, since I didn't figure out a better place... Let's go.

I'm not sure what we expect from the labels with dates. Some tickets were labeled for June 5 today (June 7). June 5 (Monday) was the first working day after Friday, when we were discussing all of this schedule...

To be honest, not too worried about this, but just to highlight the situation ;-)

**Comment by nukeador** [comment]
Yeah, sorry, June 5 was the original delivery date for all the pending stuff we already agreed before our meeting on Friday (you can see that on my emails and the [previous github issue](https://github.com/mozilla/community-development/issues/584)).

We know they are coming this week, but we want to signal that we expected them on June 5 and we need to tackle them first :-)

For the rest of the tags, June 10, June 15 and June 23, we need them to be delivered by that date.

**Comment by alpgarcia** [comment]
Firefox OS added to top 6 projects section in the report.

**Comment by alpgarcia** [comment]
We are going to double check the document latest additions and we will share it tomorrow.

**Comment by nukeador** [comment]
Thanks, it would be useful to have a small report listing the changes from the June 23 version so we can review that tomorrow during the meeting.

Cheers.

**Comment by alpgarcia** [comment]
Yep :), these are the changes:
* Added Firefox OS to ~5~ 6 projects over time section.
* Added new insight: '10% of non-employees from Firefox OS remain contributing code to other Mozilla projects after Firefox OS ended'
* Added new insight: 'Half of hired people are casual developers'
